src/Core/Content/Product/ProductEntity.php

```diff
@@ -75,6 +76,11 @@ class ProductEntity extends Entity implements \Stringable
 
     protected bool $available;
 
+    /**
+     * @deprecated tag:v6.8.0 - reason:reason:parameter-type-change - Will be native type string
+     */
+    protected ?string $type = ProductDefinition::TYPE_PHYSICAL;
```

Comment:
how can this be null? :thinking:

Reply:
Update it with an integration test, the field will has the Required flag since 6.8.0.0 but currently it doesn't

Reply:
okay, I see :+1: let's keep it like this